interfaces/tempo_cluster/v0/schema.py

```diff
@@ -0,0 +1,90 @@
+"""This file defines the schemas for the provider and requirer sides of this relation interface.
+
+It must expose two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+"""
+from enum import Enum
+from typing import Optional, Dict, Any, Literal
+
+from interface_tester.schema_base import DataBagSchema
+from pydantic import BaseModel, Field
+
+
+class TempoClusterProviderAppData(BaseModel):
+    """TempoClusterProviderAppData."""
+    tempo_config: str = Field(
+        description="The tempo configuration that the requirer should run with."
+                    "Yaml-encoded. Conform the schema that the workload version supports; "
```

Comment:
Minor: should it be `conform to/with`? https://www.merriam-webster.com/dictionary/conform

Reply:
huh. I've always thought `conform something` was also valid English. However all examples I can find are a bit on the legalese side, perhaps it's better to use explicit with/to in this context.
![image](https://github.com/canonical/charm-relation-interfaces/assets/6230162/a1ada104-74c7-4257-870e-1770e8a4d555)
will do, thanks!